vendor/service/user.go

```diff
@@ -29,8 +30,7 @@ func UserLogin(ctx iris.Context) {
 	userForm := UserFormData{}
 
 	if err := ctx.ReadForm(&userForm); err != nil {
-		ctx.StatusCode(iris.StatusUnauthorized)
-		ctx.JSON(iris.Map{
+		response.Unauthorized(ctx, iris.Map{
 			"msg":  "Unauthorized",
```

Comment:
No need to specify the whole JSON structure, but only the `data`. The functions in `response` will generate the general response structure automatically. 

```go
response.Unauthorized(ctx, iris.Map{})
```